routes/error-tracker.js

```diff
@@ -128,16 +173,30 @@ function getHandler(req, res, next) {
   if (sample > throttleRate) {
     res.set('Content-Type', 'text/plain; charset=utf-8');
     res.status(statusCodes.OK)
-      .send('THROTTLED\n')
-      .end();
+        .send('THROTTLED\n')
+        .end();
     return;
   }
 
   let exception = params.s;
+  if (ignoreMessageOrException(params.m, exception)) {
+    res.set('Content-Type', 'text/plain; charset=utf-8');
+    res.status(statusCodes.BAD_REQUEST);
+    res.send('IGNORE\n').end();
+    return;
+  }
   // If format does not end with :\d+ truncate up to the last newline.
   if (!exception.match(/:\d+$/)) {
     exception = exception.replace(/\n.*$/, '');
   }
+  exception = stackTraceConversion(exception);
+  if (!exception) {
```

Comment:
Left it here since we agreed we'll be removing the error message, sanitizing the lines and adding the error message later.